tests/ert/unit_tests/plugins/test_plugin_manager.py

```diff
@@ -115,6 +118,18 @@ def test_add_logging_handle(tmpdir):
         assert "I should write this to spam.log" in result
 
 
+def test_add_span_processor():
+    pm = ErtPluginManager(plugins=[dummy_plugins])
+    tracer_provider = TracerProvider()
+    tracer = tracer_provider.get_tracer("ert.tests")
+    pm.add_span_processor_to_trace_provider(tracer_provider)
+    with tracer.start_as_current_span("test_span"):
+        print("test_span")
```

Comment:
Maybe best to use different strings for these? You are testing the logged content of the span, not the name of the span.
Also, would it be possible to create two spans, and log to both, seeing that they would separate logging as intended?

Reply:
Changed the strings, and added an extra span. The test seeks to confirm that the add_span_processor plugin hook works, by confirming that span information is written to the span_output object of the dummy_plugins module.